.gitignore

```diff
@@ -193,3 +193,5 @@ launchSettings.json
 # include the test packages
 !test/EndToEnd/Packages/**/*
 !test/EndToEnd/ProjectTemplates/**/*.pfx
+/src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj.DotSettings
+/NuGet.sln.DotSettings
```

Comment:
Change this to ignore all files ending in `.DotSettings`, not just these two specifically.

Reply:
done